src/packages/block/block/workspace/views/edit/block-workspace-view-edit-properties.element.ts

```diff
@@ -39,25 +49,74 @@ export class UmbBlockWorkspaceViewEditPropertiesElement extends UmbLitElement {
 			this.#blockWorkspace = workspaceContext;
 			this.#setStructureManager();
 		});
+
+		// Finding the Property Editor UIs that are not allowed in blocks.
+		this.observe(
+			umbExtensionsRegistry.byTypeAndFilter('propertyEditorUi', (ui) =>
+				UMB_BLOCKS_PROPERTY_EDITOR_SCHEMAS_NOT_SUPPORTED.includes(ui.meta.propertyEditorSchemaAlias ?? ''),
+			),
+			(manifests) =>
+				(this.#disallowedPropertyEditorUis = manifests.map((manifest) => ({
+					uiAlias: manifest.alias,
+					schemaAlias: manifest.meta.propertyEditorSchemaAlias!,
+				}))),
+		);
 	}
 
 	#setStructureManager() {
 		if (!this.#blockWorkspace || !this.#managerName) return;
 		this.#propertyStructureHelper.setStructureManager(this.#blockWorkspace[this.#managerName].structure);
 		this.observe(
 			this.#propertyStructureHelper.propertyStructure,
-			(propertyStructure) => {
+			async (propertyStructure) => {
+				await this.#checkForDisallowed(propertyStructure);
 				this._propertyStructure = propertyStructure;
 			},
 			'observePropertyStructure',
 		);
 	}
 
+	async #checkForDisallowed(properties: Array<UmbPropertyTypeModel>) {
+		const { data } = await this.#dataTypeItemRepository.requestItems(
```

Comment:
I have been reading the code of this PR, and I would suggest to move this logic into ´umb-property-type-based-property´.

I think the approach would to make the ability of parsing a list of `unsupported-property-editor-schemas`.

As the umb-property-type-based-property already knows its data-type and the extension manifest then it does not need to do retrieve any data to make this happen. Also because I'm not fond of requesting the data type here just to check if its supported. Also the amount of code for this is quite a lot, and that is bad for maintainability and readability.

This would as well make it possible to easily implement similar in other places and ensure consistency.

A side note, is it really not possibly for us to support these, I really hopped the architecture for the temp files would enable us to support this.